critiquebrainz/db/review.py

```diff
@@ -684,3 +684,28 @@ def reviewed_entities(*, entity_ids, entity_type):
         })
         reviewed_ids = [str(row[0]) for row in results.fetchall()]
     return reviewed_ids
+
+def get_top_contributors(limit=10):
+    query = sqlalchemy.text("""
```

Comment:
We prefer docstrings for any function we add to the codebase. Have a look at other functions for your reference.

Reply:
The docstrings we use are google style docstrings, documented here: https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings

Reply:
Okay, thank u for the references, i've added. :smile: